lib/crypt_keeper/model.rb

```diff
@@ -85,7 +85,11 @@ def search_by_plaintext(field, criteria)
       # Public: Encrypt a table for the first time.
       def encrypt_table!
         enc       = encryptor_klass.new(crypt_keeper_options)
-        tmp_table = Class.new(ActiveRecord::Base).tap { |c| c.table_name = self.table_name }
+
+        tmp_table = Class.new(ActiveRecord::Base).tap do |c|
+          c.table_name = self.table_name
+          c.inheritance_column = :type_disabled
```

Comment:
@jmazzi this is just a way to disable STI. You set it to a non-existant column and it will be disabled.